packages/space-opera/src/components/model_viewer_snippet/reducer.ts

```diff
@@ -64,3 +58,30 @@ export function extraAttributesReducer(state: any = {}, action: Action): any {
       return state;
   }
 }
+
+const SET_HEIGHT = 'SET_HEIGHT';
+export function dispatchHeight(height: number) {
+  return {type: SET_HEIGHT, payload: height};
+}
+
+const SET_MIMETYPE = 'SET_MIMETYPE';
+export function dispatchMimeType(type: ImageType) {
```

Comment:
I put this in because I thought I'd allow choosing PNG, JPEG, WEBP, but then I realized WEBP is definitely the best, so I removed the UX. I left the logic in case it's useful down the road.